compiler/luci/import/src/Importer.cpp

```diff
@@ -122,12 +122,19 @@ void convert_graph(const luci::GraphBuilderSource &source, luci::CircleReader &r
   }
 
   // Create CircleConst nodes for constant tensors.
+  // Create CircleNodes for constant tensors.
   // NOTE Origin is intentionally not provided for constants.
+  auto const_builder = source.lookup(luci::NodeBuilderType::BUFFER);
+  if (not const_builder)
+    throw oops::UserExn("Not supported", "tensor with buffer builder");
+
   for (uint32_t i = 0; i < tensors.size(); ++i)
   {
-    luci::CircleConst *const_node = luci::create_circleconst(&gb_context, i);
+    auto *const_node = const_builder->build(i, &gb_context);
```

Comment:
Q) why changed to auto?

Reply:
We can't use `luci::CircleConst` here because `build()` method returns `luci::CircleNode`. As another option we can change to `luci::CircleNode` instead `auto` but `auto` looks better for me.